internal/service/advancedclustertpf/move_state_test.go

```diff
@@ -8,9 +8,11 @@ import (
 	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
 	"github.com/hashicorp/terraform-plugin-testing/tfversion"
 	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/unit"
 )
 
 func TestAccAdvancedCluster_moveBasic(t *testing.T) {
+	unit.SkipInReplayMode(t)
```

Comment:
as discussed offline we can better use a prefix for test candidates like TestAccMockable

Reply:
Changed in 9fc4d8be039bb5d12205dcc12d3ff906444013f3